src/features/swap/index.tsx

```diff
@@ -61,6 +55,22 @@ const SwapWidget = ({ sell }: Params) => {
   const wallet = useWallet()
   const { isConsentAccepted, onAccept } = useSwapConsent()
 
+  const appData: SafeAppData = useMemo(
+    () => ({
+      id: 1,
+      url: 'https://app.safe.global',
+      name: SWAP_TITLE,
+      iconUrl: darkMode ? './images/common/safe-swap-dark.svg' : './images/common/safe-swap.svg',
+      description: 'Safe Apps',
+      chainIds: ['1', '100'],
+      accessControl: { type: SafeAppAccessPolicyTypes.NoRestrictions },
+      tags: ['safe-apps'],
+      features: [SafeAppFeatures.BATCHED_TRANSACTIONS],
+      socialProfiles: [],
+    }),
+    [darkMode, tradeType],
```

Comment:
```suggestion
    [darkMode],
```

the eslint warning seems correct here.